crates/aranya-daemon/src/aranya.rs

```diff
@@ -738,6 +743,36 @@ where
         })
         .in_current_span()
     }
+
+    /// Query assigned network identifiers.
+    #[allow(clippy::type_complexity)]
+    #[instrument(skip_all)]
+    #[cfg(feature = "afc")]
+    fn query_afc_network_names(
```

Comment:
We are using the `off_graph` naming convention for ephemeral session commands:
```suggestion
    fn query_afc_network_names_off_graph(
```